src/parameterized_tendencies/microphysics/precipitation.jl

```diff
@@ -163,6 +160,17 @@ function precipitation_tendency!(
                     ᶜts[colidx],
                     ᶜΦ[colidx],
                 )
+            if p.atmos.surface_model isa PrognosticSurfaceTemperature
```

Comment:
Same as before, it might be good to always calculate this.

Reply:
Does it mean that if we are running with `! PrognosticSurfaceTemperature` we don't expect energy conservation?

Reply:
You could do it. The fluxes are just not being tracked / collected here. We can add it to the next PR on conservation.